build.gradle.kts

```diff
@@ -52,7 +52,7 @@ repositories {
 extra.apply {
     set("annotationsVersion", "22.0.0")
     set("mongodbDriverVersion", "[4.5.0,4.5.99)")
-    set("sparkVersion", "3.1.2")
+    set("sparkVersion", "3.2.2")
```

Comment:
Would allow to set it using a sys prop, to be able to cross build.

Reply:
Would also add a similar `scalaBinaryVersion` to allow to cross build between 2.1X.
This should also be use as suffix for the artifactId in publication, as this lib in not a pure Java one, but a Scala one depending on Scala library version which can be cross built accordingly.

Reply:
Yea,
I've already mention this point in another pr (https://github.com/mongodb/mongo-spark/pull/58#issuecomment-1219430851).
To have cross build, we need to have specifics sources for 2.12 and 2.13 (implicits conversion from java to scala in src/main/java/com/mongodb/spark/sql/connector/schema/BsonDocumentToRowConverter.java) and i'm not a gradle power user to do that :D, 
i'm more comfortable with sbt do to that